widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.java

```diff
@@ -155,6 +157,19 @@ public synchronized static void init(GliaWidgetsConfig gliaWidgetsConfig) {
         Dependencies.getGliaThemeManager().applyJsonConfig(gliaWidgetsConfig.uiJsonRemoteConfig);
     }
 
+    /**
+     * Retrieves an instance of {@link EngagementLauncher}.
+     *
+     * @param queueIds A list of queue IDs to be used for the engagement launcher.
+     *                 When null, the default queues will be used.
+     * @return An instance of {@link EngagementLauncher}.
+     * @throws IllegalStateException if the SDK is not initialized.
```

Comment:
I thought that GliaException was smth internal in this case because it is mostly handled inside the Widgets SDK. So, in this case, it may add a dependency from the core in public APIs.
What do you think about my concerns?